CommunityFridgeMapApi/tests/unit/test_image.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import base64
+from unittest.mock import patch, ANY
+from functions.image.v1.app import ImageHandler
+from tests.assert_resposne import assert_response
+from tests.storage_stub import StorageStub
+
+
+def test_upload(storage_stub: StorageStub):
+    blob = b'123123123'
```

Comment:
Couple questions:
1. I did not use `unittest.TestCase` here because pytest fixture is not [supported](https://docs.pytest.org/en/7.1.x/how-to/unittest.html#pytest-features-in-unittest-testcase-subclasses) in unittest. Is this okay?
2. I've only written a happy-path case here. Let me know if we need to add any particular test cases.

Reply:
Yeah it's fine to not use unittest.TestCase as long as it still gets run